### ISSUE TITLE
help

### ISSUE THREAD

**Issue by clarinetandmonkey** [dnglokpor/IsekaiBot#1]
@dnglokpor 
I was using a snippet of this code for a project I was working on, but i cannot find the definition of `winChances`

```python 
win = winChances(gang, boss, 1)

can i have the definition of winChances please?

**Comment by clarinetandmonkey** [comment]
I believe it's just escape chances but I copy and paste it and change it to winChances?

**Comment by dnglokpor** [comment]
@clarinetandmonkey thanks for the question. I realize I changed the way I was doing the encounters later because the way I was modeling the win chances wasn't really good. So I changed the display to show damage dealt and damage taken instead and let the player decide whether he could win or not. But it seems I forgot to make the changes in the code for the tutorial. refer to **encounter( )** method at line **1824** for the new implementation. I'll update the tutorial code as well.
